.devcontainer/docker-compose.yml

```diff
@@ -3,18 +3,18 @@ version: "3.8"
 services:
   dev:
     build:
-      context: .
-      dockerfile: Dockerfile
+      context: ..
+      dockerfile: /.devcontainer/Dockerfile
```

Comment:
Probably just a typo, the `/` at the beginning is making VS Code look in the root of the filesystem for a .devcontainer/Dockerfile 😅 

Should be just `.devcontainer/Dockerfile`